landing_page/templates/landing_page/base.html

```diff
@@ -46,6 +48,7 @@ <h1>ReeldIn</h1>
   <nav>
     <a href="index.html">Home</a>
     <a href="about.html">About</a>
+    <a href="movie.html">Movie Info</a>
```

Comment:
The movie info page can only be accessed by clicking on a movie through the homepage, search, or when it's recommended so it won't be in the navigation.